src/View/ViewableData.php

```diff
@@ -293,7 +293,7 @@ public function castingHelper($field)
         }
 
         // Fall back to default_cast
-        $default = $this->config()->get('default_cast');
+        $default = self::config()->get('default_cast');
```

Comment:
It should be static::config(), but $this->config() is still fine. Apparently calling static methods as instance methods isn't strictly bad-practice, kind of like negative css margins. :)